Content/ContentResolver.php

```diff
@@ -23,12 +24,9 @@ class ContentResolver implements ContentResolverInterface
      */
     private $resolvers;
 
-    /**
-     * @param ContentTypeResolverInterface[] $resolvers
-     */
-    public function __construct(array $resolvers)
+    public function __construct(RewindableGenerator $resolvers)
```

Comment:
do you need to use `RewindableGenerator` here? how did you find the type? 🙂 

if not, would prefer to use `iterable` as described in the docs. that should make testing easier too: https://symfony.com/doc/4.4/service_container/tags.html#reference-tagged-services

Reply:
It is not really possible